libgdf/include/gdf/cffi/functions.h

```diff
@@ -194,6 +194,12 @@ gdf_error gdf_segmented_radixsort_generic(gdf_segmented_radixsort_plan_type *hdl
                                      unsigned *d_begin_offsets,
                                      unsigned *d_end_offsets);
 
+
+// transpose
+gdf_error gdf_transpose(size_t ncols,
```

Comment:
Please add documentation that's inline with the template here: https://github.com/rapidsai/cudf/blob/master/libgdf/src/example_documentation.cpp